metricbeat/docs/modules/openmetrics/collector.asciidoc

```diff
@@ -9,6 +9,7 @@ beta[]
 
 include::../../../module/openmetrics/collector/_meta/docs.asciidoc[]
 
+This is a default metricset. If the host module is unconfigured, this metricset is enabled by default.
```

Comment:
Maybe this is not accurate? Leftover from a copy paste maybe?
```suggestion
This is a default metricset. If the openmetrics module is unconfigured, this metricset is enabled by default.
```

Reply:
Changed `host` to `openmetrics`

Reply:
@ChrsMark `host` is hardcoded in template, it can not be changed. That's why build is failing now.
https://github.com/elastic/beats/blob/master/metricbeat/scripts/mage/template/metricsetDoc.tmpl#L16

Reply:
Ah I see, sorry for the noise then :).